docs/tutorials/data/ios/config.txt

```diff
@@ -22,6 +22,7 @@ interface FastEthernet4
    shutdown
    switchport mode trunk
    switchport trunk allowed vlan 10,20
+   loopback mac
```

Comment:
interesting, never seen that command before. What does it do? Do you have by any chance a link to the documentation? Couldn't find anything :(

Reply:
https://www.cisco.com/c/en/us/td/docs/interfaces_modules/shared_port_adapters/configuration/7600series/sipspasw/76tbleth.pdf
Section: Configuring the Interface for Internal Loopback
Guess my google fu is a bit better :P